stable/pipelines/values.yaml

```diff
@@ -504,7 +504,7 @@ pipelines:
         defaultExchange: {{ .Values.rabbitmq.root_vhost_exchange_name }}
         amqpVhost: {{ .Values.rabbitmq.build_vhost_name}}
         amqpRootVhost: {{ .Values.rabbitmq.root_vhost_name }}
-        protocol: amqps
+        protocol: {{ .Values.rabbitmq.protocol | default "amqps" }}
```

Comment:
Add this to the values instead of using | default.
This causes redundant duplications

Reply:
@danielezer , This is when we are using an external rabbitmq. If we removed the default (amqps) which was there before. would that break any existing deployments , if they dont provide the protocol. ?

Reply:
@rahulsadanandan I'm not asking you to remove the protocol. I'm asking that you add `rabbitmq.protocol=amqps` to the values.yaml and that will remove the need to use `| default "amqps"`

Reply:
Changed.